torch/serialization.py

```diff
@@ -167,6 +169,34 @@ def set_default_load_endianness(endianness):
     _default_load_endian = endianness
 
 
+_compute_crc32: bool = True
+
+
+def get_default_crc32_options() -> bool:
+    """
+    Get whether :func:`torch.save` computes and writes crc32 for each record.
+
+    Defaults to ``True``.
+    """
+    return _compute_crc32
+
+
+def set_default_crc32_options(compute_crc32: bool):
```

Comment:
nit: why `set_default_crc32_options` vs `set_crc32_options` ? There is no way to override this flag from the end user of torch.save point of view.

Reply:
will use `set_crc32_options`